### PR TITLE
New Redis namespace

### PR DESCRIPTION
**Pull request by maksimf**

### CODE REVIEW

**Comment by maksimf** [comment]
@patricioe This PR is ready

File: lib/splitclient-rb/split_config.rb [hunk 1]
```diff
@@ -249,6 +251,10 @@ def self.default_redis_url
       'redis://127.0.0.1:6379/0'
     end
 
+    def self.default_redis_namespace
+      "SPLITIO/ruby-#{VERSION}"
```
Comment (patricioe) [inline lib/splitclient-rb/split_config.rb]: I can't find where VERSION is coming from.

Comment (maksimf) [inline lib/splitclient-rb/split_config.rb]: This constant comes from the class in the current namespace (i.e. `SplitIoClient::VERSION`) which is defined here: `lib/splitclient-rb/version.rb`

**Comment by patricioe** [comment]
there are conflicts to merge @maksimf

**Comment by maksimf** [comment]
@patricioe resolved

**Pull request merged**